pkg/graph/schema.graphql

```diff
@@ -1362,6 +1362,35 @@ type SystemIntakeContactsPayload {
   invalidEUAIDs: [String!]!
 }
 
+"""
+The inputs to the user feedback form
+"""
+input SendFeedbackEmailInput {
+  isAnonymous: Boolean!
+  easiServicesUsed: [String!]
```

Comment:
Small thing - this array should be non-nullable, as well as its contents, i.e. `[String!]!`.